code/controllers/subsystems/payment.dm

```diff
@@ -108,9 +121,12 @@ SUBSYSTEM_DEF(payment_controller)
 			message.title = "[GLOB.using_map.station_name] Automated Payroll System"
 
 			if(!H.mind.pay_suspended)
-				message.stored_data = "[H.real_name],\n\n<b>[economic_modifier] Th</b> has been deposited into your account following hourly payroll payouts.\nThank you for your continued hard work.\n\n<i>This message is brought to you by [sponsor] - [this_slogan]</i>"
-			else
-				message.stored_data = "[H.real_name],\n\nAs your wages have been suspended, your payout has not been processed. Please direct all complaints to your payroll administrator."
+				if(H.mind.assigned_role == "Passenger")
+					message.stored_data = "[H.real_name],\n\n<b>[passenger_fee] Th</b> has been deducted from your account as part of the hourly passenger fee.\nThank you for travelling with us.\n\n<i>This message is brought to you by [sponsor] - [this_slogan]</i>"
+				else if(!H.mind.pay_suspended)
+					message.stored_data = "[H.real_name],\n\n<b>[economic_modifier] Th</b> has been deposited into your account following hourly payroll payouts.\nThank you for your continued hard work.\n\n<i>This message is brought to you by [sponsor] - [this_slogan]</i>"
+				else
+					message.stored_data = "[H.real_name],\n\nAs your wages have been suspended, your payout has not been processed. Please direct all complaints to your payroll administrator."
```

Comment:
The final `else` statement is unreachable. You'll need to remove the `pay_suspended` check on LN123 and nudge everything an indent in.